vtrace/notifiers.py

```diff
@@ -42,9 +42,10 @@ def handleEvent(self, event, trace):
     def notify(self, event, trace):
         print("Got event: %d from pid %d" % (event, trace.getPid()))
 
+
 class VerboseNotifier(Notifier):
     def notify(self, event, trace):
-        print("PID %d - ThreadID (%d) got" % (trace.getPid(), trace.getMeta("ThreadId")))
+        print "PID %d - ThreadID (%d) got" % (trace.getPid(), trace.getMeta("ThreadId")),
```

Comment:
you can leave the parens.
`
print("PID....."),
`
should work fine

Reply:
and @invisig0th 's been pushing for a close to py3-compatible as possible, at least wrt print's